packages/hdwallet-provider/test/typings.test-d.ts

```diff
@@ -0,0 +1,4 @@
+import { expectType } from "tsd";
```

Comment:
@cds-amal and I discussed and it seems there will need to be some more work involved with testing these changes.